Classes/WPEditorFormatbarView.m

```diff
@@ -92,6 +92,20 @@ - (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection
     DDLogInfo(@"Format bar trait collection did change from: %@", previousTraitCollection);
 }
 
+- (void)layoutSubviews {
+    [super layoutSubviews];
+    // HACK: Sergio Estevao (2017-10-10): Change the size of the items whe running on a device with a width smaller or equal than 320 (iPhone SE)
+    if (self.frame.size.width <= 320) {
+        for (UIBarButtonItem *item in self.leftToolbar.items) {
+            item.width = roundf(item.image.size.width * 0.75);
+        }
+    }
+
+    if( @available(iOS 11, *)) {
```

Comment:
@SergioEstevao For posterity, could you put a quick comment here about why this is used?